src/app/application.cpp

```diff
@@ -111,7 +111,6 @@ Application::Application(const QString &id, int &argc, char **argv)
 {
     qRegisterMetaType<Log::Msg>("Log::Msg");
 
-    setApplicationName("qBittorrent");
```

Comment:
This must be an artifact from rebasing... thanks for catching it.
For me, it seems it's more logical to remove the second function call if there is no downside.

Reply:
None that I know of. Will update the PR.